build/push-official-release.sh

```diff
@@ -22,7 +22,7 @@ set -o pipefail
 
 KUBE_RELEASE_VERSION=${1-}
 
-VERSION_REGEX="v[0-9]+.[0-9]+(.[0-9]+)?"
```

Comment:
Hmmm, looks like the third part was optional at some point... Maybe that should be kept? What if v1.0 is just v1.0 and not v1.0.0? Should be OK to just keep the `?` and check for an empty `${BASH_REMATCH[3]}` and replace it by 0 in that case.